evalml/utils/gen_utils.py

```diff
@@ -270,3 +270,27 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+def _convert_woodwork_types_wrapper(pd_data):
+    """
+    Converts a pandas data structure that may have extension or nullable dtypes to dtypes that numpy can understand and handle.
+
+    Arguments:
+        pd_data (pd.Series, pd.DataFrame, pd.ExtensionArray): Pandas data structure
+
+    Returns:
+        New pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
+    """
+    nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64',
+                                 pd.BooleanDtype: 'bool',
+                                 pd.StringDtype: 'object'}
```

Comment:
So we don't need an entry for floats?

Reply:
We will later, but Float64Dtype is not out yet! I think @gsheni mentioned it'll be released in pandas 1.2.0 (https://github.com/alteryx/evalml/pull/1299#discussion_r505020442)